src/pages/workspaces/workspace/workflows/WorkflowView.js

```diff
@@ -762,39 +776,87 @@ const WorkflowView = _.flow(
                 ])
             ])
           ]),
-          div({ style: { marginTop: '1rem' } }, [
-            h(LabeledCheckbox, {
-              disabled: currentSnapRedacted || !!Utils.computeWorkspaceError(ws),
-              checked: useCallCache,
-              onChange: v => this.setState({ useCallCache: v })
-            }, [' Use call caching']),
-            span({ style: { margin: '0 0.5rem 0 1rem' } }, [
-              h(LabeledCheckbox, {
-                checked: deleteIntermediateOutputFiles,
-                onChange: v => this.setState({ deleteIntermediateOutputFiles: v }),
-                style: { marginLeft: '1rem' }
-              }, [' Delete intermediate outputs'])
-            ]),
-            h(InfoBox, [
-              'If the workflow succeeds, only the final output will be saved. Subsequently, call caching cannot be used as the intermediate steps will be not available. ',
-              h(Link, {
-                href: 'https://support.terra.bio/hc/en-us/articles/360039681632',
-                ...Utils.newTabLinkProps
-              }, ['Click here to learn more.'])
-            ]),
-            span({ style: { margin: '0 0.5rem 0 1rem' } }, [
+          div([
+            // Because the retries numeric input is not always visible, and will be taller than the other components
+            // on this line, wrap the components that are always visible with a div and add top/bottom margins to
+            // avoid vertical resizing when the numeric input is rendered.
+            div({ style: { display: 'inline-block', marginTop: '0.5rem', marginBottom: '0.5rem' } }, [
```

Comment:
To avoid wrapping and the height change I think we can do:
```suggestion
div({ style: { display: 'flex', alignItems: 'center', minHeight: '2.5rem', minWidth: 'max-content', margin: '0.5rem 0' } }, [
```

With this you could remove the `inline-block` styling and push the conditional render into this div block

Reply:
@petesantos I can't figure out a way to use 'center' for alignItems and keep the checkbox text aligned properly vertically (and in comparison to the input label). I pushed up a change for you to consider that puts all the checkboxes within a span with top/bottom margin spacing so that I can use 'baseline' for alignItems.